### PR TITLE
Deploy fix

### PR DESCRIPTION
**Pull request by onyijne**

### CODE REVIEW

File: src/views/Jobs.vue [hunk 1]
```diff
@@ -25,7 +25,13 @@
           />
         </div>-->
         <div class="relative md:w-1/4">
-          <Select place-holder="Category" :options="categories" v-model="filter.category" />
+          <Input
+            v-model="filter.category"
```
Comment (techadam) [inline src/views/Jobs.vue:29]: Why was the options props removed here?

File: src/views/Jobs.vue [hunk 2]
```diff
@@ -109,9 +114,9 @@ const enums = computed(() => {
   return store.state.global.EnumDetails;
 });
```
Comment (techadam) [inline src/views/Jobs.vue:116]: Why was this commented out? Is there another  way we are getting the categories here?

**Review by techadam — CHANGES_REQUESTED** [review]

File: src/views/Jobs.vue (hunk 1 above)
Comment (onyijne) [reply src/views/Jobs.vue:29]: it should be a search box

File: src/views/Jobs.vue (hunk 2 above)
Comment (onyijne) [reply src/views/Jobs.vue:116]: not being used.

**Review by techadam — APPROVED** [review]

**Pull request merged**